ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/management-security.cfg

```diff
@@ -26,6 +26,15 @@ management security
       minimum upper 5
       maximum repetitive 6
       maximum sequential 7
+   session shared-secret profile profile0
```

Comment:
I think we are missing a ! before the first one?

Reply:
if we do not configure the commands present above session shared-secret. It is generating ! after management security. So added ! after every profile. 

management security
 !
session shared-secret profile profile0

Reply:
We normally just add it above the iterative models. Having an extra ! is ok for now. We will probably build some filter to generate these correctly across the board.

Reply:
In your current logic we are missing the ! above the first profile when you have the other commands.

Reply:
Fixed.